### PR TITLE
Rename geo-compose -> geoview-compose

### PR DESCRIPTION
**Pull request by prupani-7**
## Description

PR to rename geo-compose -> geoview-compose

## Links and Data

Sample Epic: `runtime/kotlin/issues/3653`

## What To Review

- Leaving out the renaming changes in tools/NewModuleScript to Ruiqi, as she will address that in [#3649]
- Review the refactoring changes elsewhere

## How to Test

Run the affected sample on the sample viewer or the repo.

### CODE REVIEW

**Comment by prupani-7** [comment]
Seems like the style checker errors are false positives?
Folder path: analyze-hotspots
1. analyze-hotspots - Error tags - Tags are not sorted.
2. analyze-hotspots - Error checking extra tags due to previous error

The readme tags seem sorted, and so do the keywords in metadata json file

File: analyze-hotspots/README.md [hunk 1]
```diff
@@ -30,8 +30,8 @@ Tap on Analyze, and select a date from the "FROM" DatePicker and "TO" DatePicker
 
 ## Additional information
 
-This sample uses the GeoCompose Toolkit module to be able to implement a Composable MapView.
+This sample uses the GeoViewCompose Toolkit module to be able to implement a Composable MapView.
 
 ## Tags
 
-analysis, density, geocompose, geoprocessing, hot spots, hotspots, toolkit
+analysis, density, geoviewcompose, geoprocessing, hot spots, hotspots, toolkit
```
Comment (ruiqima) [inline analyze-hotspots/README.md:37]: Style check shows the tags needs to be reordered here, should be `geoprocessing, geoviewcompose`

**Review by ruiqima — APPROVED** [review]
One comment about tag reordering, others looks good 👍 Tested with sample viewer, it built successfully. 

Context for other reviewers: updates related to NewModuleScript will be handled in a separate issue, as there requires some other changes too.

**Comment by prupani-7** [comment]
Thanks Ruiqi, I easily overlooked that!

**Review by eri9000 — COMMENTED** [review]
Good work @prupani-7 
I see some more references to `geocompose` in `MainScreenTemplate.kt` and in `.github/.../Main.yaml` [here](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/blob/main/.github/workflows/main.yml). 
do these need to be updated?

**Comment by prupani-7** [comment]
> Good work @prupani-7 I see some more references to `geocompose` in `MainScreenTemplate.kt` and in `.github/.../Main.yaml` [here](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/blob/main/.github/workflows/main.yml). do these need to be updated?

as mentioned above the MainScreenTemplate.kt update within the NewModuleScript folder will be handled by Ruiqi's PR.

And the Main.yml has just a mention about the branch name which is  `feature-branch/geo-compose` 
it doesn't reference the toolkit name.

**Review by eri9000 — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**